### PR TITLE
docs: update PR template

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Daniel, here's an update to your PR #12. The info will be there as a regular part of the workflow, so I recommend that we make a new section for it. 

The other places that I mention PRs and workflows is in the docs. Check out the changes at

- [GitHub Action workflows](https://pages.github.ibm.com/allen-dean/tim-docs/#/gh-actions)
- [Commit your code and submit for review](https://pages.github.ibm.com/allen-dean/tim-docs/#/contribute-module?id=commit-your-code-and-submit-for-review)

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version. 

Your notes helps the merger write the commit message for the PR that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -25,14 +25,19 @@ Identify the type of release. For information about the changes in a semantic ve
...
 
-Your notes helps the merger write the commit message for the PR that is published in the release notes for the module.
+Your notes help the merger write the commit message for the PR that is published in the release notes for the module.
 
----
+### Run the pipeline
 
-> **Note**  
-If the pipeline does not automatically trigger it has been migrated to the new v2 pipeline system a `Collaboator` must post the exact comment `/run pipeline`. Only trigger a pipeline once you are happy with the PR and you are sure the tests should pass.
+If the CI pipeline doesn't run when you create the PR, the module requires a user with GitHub collaborators access to run the pipeline.
```
Comment (daniel-butler-irl) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not sure about the wording `the module requires` @ocofaigh any thoughts?

Comment (ocofaigh) [reply .github/PULL_REQUEST_TEMPLATE.md]: replace `the module` with the `the PR` ?

**Comment by SirSpidey** [comment]
@daniel-butler-irl this PR is built off and merges into your PR 12. Do you want me to create another PR to main, or do you want to reopen yours, merge this, and and then merge yours?

**Comment by daniel-butler-irl** [comment]
Sorry missed that. Let's create a PR directly to the main. No point merging to mine first it's adding nothing.

**Comment by SirSpidey** [comment]
closed in favor of #14

**Pull request closed without merging**